property_panel/reducers/intentPropertiesPanel.js

```diff
@@ -18,13 +18,13 @@ const intentPropertiesPanel = (state = {
           userSays: {
             hideProperty: action.nodeType != 'userSays',
             text: action.targetNode.data.user_says || "", 
-            selectedNode: parseInt(action.targetNode.data.id)
+            selectedNode: parseInt(action.targetNode.data.id),
           },
           response: {
             hideProperty: action.nodeType != 'response',
             text: action.targetNode.data.response || "", 
             selectedNode: parseInt(action.targetNode.data.id),
-            action: action.targetNode.data.action
+            action: action.targetNode.data.action || ""
```

Comment:
targetNode could be user_says or response. If it is response node then there is no action data. For same reason to write action.targetNode.data.user_says || "" and action.targetNode.data.response || ""
There is no problem displayed on window without it but where is error at console
It could be not a good design...maybe I can try to fix it